.github/scripts/end2end/run-e2e-test.sh

```diff
@@ -120,7 +121,36 @@ run_e2e_test() {
         --env=MONGO_WRITE_CONCERN=${MONGO_WRITE_CONCERN} \
         --env=MONGO_AUTH_USERNAME=${MONGO_AUTH_USERNAME} \
         --env=MONGO_AUTH_PASSWORD=${MONGO_AUTH_PASSWORD} \
-        --command -- sh -c "${2}"
+        --env=MOCHA_FILE=${MOCHA_FILE} \
+        --override-type strategic \
+        --overrides='
+{
+  "apiVersion": "v1",
+  "kind": "Pod",
+  "spec": {
+    "containers": [
+      {
+        "name": "'$POD_NAME'",
+        "volumeMounts": [
+          {
+            "name": "artifacts",
+            "mountPath": "/artifacts"
```

Comment:
for readability, it may be better to use a different name for this "inner" path : maybe `reports`, so that we can easily see the different between the host `/artifacts` and the inner mount point `/reports`

→ in the runner, the reports would thus be in `/artifacts/data/reports`